### DISCUSSION TITLE
How to use LQR to control TMD?

### DISCUSSION THREAD

**Discussion by icnosvbovwb** [NatLabRockies/ROSCO#285] [Q&A]
Dear @jjonkman
Can you tell me how to actively control TMD with LQR? The TMD is place on the platform to reduce platform pitch. But I am not familiar with LQR, how do I use it?

**Comment by dzalkind** [comment]
Hi, 

Jason typically answers questions on the OpenFAST GitHub or NWTC forum.

TMDs can be controlled via the StructControl module of OpenFAST.  A simple example of actively controlling structural controllers can be found here: https://github.com/NREL/ROSCO/blob/main/Examples/23_structural_control.py 

However, no specific controller is determined and it's up to the user to write their own methods, like LQR.

Best, Dan
